site/src/comparison.rs

```diff
@@ -798,21 +791,21 @@ impl Comparison {
             .partition(|s| category_map.get(&s.benchmark()) == Some(&Category::Primary));
 
         let (primary_errors, secondary_errors) = self
-            .new_errors
+            .newly_failed_benchmarks
             .into_iter()
             .partition(|(b, _)| category_map.get(&b.as_str().into()) == Some(&Category::Primary));
 
         let primary = Comparison {
             a: self.a.clone(),
             b: self.b.clone(),
             statistics: primary,
-            new_errors: primary_errors,
+            newly_failed_benchmarks: primary_errors,
```

Comment:
It seems to me that the partition here is now useless, since the summary doesn't even access the fields? But on the other hand it keeps the comparisons in a consistent state, although it's not used at this moment.

Reply:
Yes, I have an idea how to clean this up, but I'd like to do it in a separate PR.